pages/statistics.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box } from "@mui/material";
+import config from "config";
+import { PostDocumentWithoutBody } from "interfaces";
+import Head from "next/head";
+
+import Footer from "@/components/Footer";
+import Header from "@/components/Header";
+import Intro from "@/components/Intro";
+import StandWithUkraine from "@/components/StandWithUkraine";
+import StatisticPage from "@/components/Statistic";
+
+import client from ".tina/__generated__/client";
+
+interface Props {
+  posts: PostDocumentWithoutBody[];
+}
+
+const containterStyles = {
+  mx: "auto",
+  px: "15px",
+  ["@media(min-width: 768px)"]: {
+    width: "750px",
+  },
+  ["@media(min-width: 1020px)"]: {
+    width: "1000px",
+  },
+  ["@media(min-width: 1260px)"]: {
+    width: "1230px",
+  },
```

Comment:
Do you need so many breakpoints in here?

Can the page just be fluid / flexible?